src/components/tasks/hiddenInfoTask/hidden-problem-shared.js

```diff
@@ -25,12 +25,27 @@ const SubmitElementsButton = styled.input`
     ${'' /* border-radius: 6px; */}
 `;
 
+const ReadyButton = styled.input`
+    background-color: #66e29a;
+    border: none;
+    cursor: pointer;
+    width: 80%;
+    padding: 1rem;
+    margin: 0.5rem;
+    font-size: 15px;
+    white-space: normal;
+    align-self: center;
+    margin-top: 2rem;
+    border-radius: 6px;
+`;
+
 function HiddenProblem({ submittable }) {
     const { state } = useLocation();
     const { meetingId, netId, participantId } = state;
     const navigate = useNavigate();
     const location = useLocation();
     const [choice, setChoice] = useState([]);
+    const [allReady, setAllReady] = useState(false);

```

Comment:
I'm thinking we should maybe call this state something else, because right now in the DB, type: 'ready' refers to the participants who've finished the individual reading component of the hidden task.

Maybe we can consider calling this state 'groupReady' instead -- this way the type is 'groupReady' instead of 'ready' and it won't be conflated with the submitReady call that participants do after finishing the indiv hidden task. It would also better align with 'groupChoices' vs 'choices' to indicate if the choice submission is for the indiv or the group task.